README.md

```diff
@@ -18,6 +18,7 @@ For more details see articles:
 
 - Fallback composition - a single `<style>:host{display: block;}</style><slot></slot>` to display all things from light DOM in case no composition was provided
 - Default composition - composition (or multiple concatenated compositions) found in the partial HTML view (`<template is="declarative-shadow-dom">` part)
+- Host composition - composition found in the host HTML view (content of `<starcounter-include ...><template is="declarative-shadow-dom" mode="host">`). It's used to enforce by host view a given composition for any kind of elements.
```

Comment:
On the today's meeting with @tomalec, we decided to call this `presentation="parent"`